ibtk/lib/Makefile.am

```diff
@@ -354,12 +355,15 @@ pkg_include_HEADERS += \
 ../include/ibtk/private/PETScSAMRAIVectorReal-inl.h \
 ../include/ibtk/private/StreamableManager-inl.h
 
+## TODO: Should JacobianCalculator.h be in include or src?
+
```

Comment:
This isn't relevant to the PR - lets open an issue.

Reply:
#969

Reply:
Removed comment from that `Makefile.am`.